### ISSUE TITLE
SetSrgbcBC[1-3] Not Defined

### ISSUE THREAD

**Issue by Lin20** [GPUOpen-Tools/compressonator#189]
The functions
```CPP
int CMP_CDECL SetSrgbBC1(void* options, bool sRGB);
int CMP_CDECL SetSrgbBC2(void* options, bool sRGB);
int CMP_CDECL SetSrgbBC3(void* options, bool sRGB);
```
are declared in `cmp_core.h`, but don't appear to be defined anywhere. After skimming around, it looks like sRGB is supported in the actual compression kernels, though I haven't tested it after just manually assigning the options directly.

Thank you for this awesome library and all the work that goes into it!

**Comment by chloekek** [comment]
`SetSrgbBC{2,3}` are defined in `cmp_core/shaders/bc{2,3}_encode_kernel.cpp` respectively, but a definition for `SetSrgbBC1` is indeed nowhere to be found.

**Comment by NPCompress** [comment]
The issue has been resolved in current release v4.4
